webdriver/tests/bidi/script/evaluate/evaluate.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget, ScriptEvaluateResultException
+from ... import recursive_compare, any_string
+
+
+@pytest.mark.asyncio
+async def test_eval(bidi_session, top_context):
+    result = await bidi_session.script.evaluate(
+        expression="1 + 2",
+        target=ContextTarget(top_context["context"]))
+
+    assert result == {
+        "type": "number",
+        "value": 3}
+
+
+@pytest.mark.asyncio
+async def test_params_expression_invalid_script(bidi_session, top_context):
+    with pytest.raises(ScriptEvaluateResultException) as exception:
+        await bidi_session.script.evaluate(
+            expression='))) !!@@## some invalid JS script (((',
+            target=ContextTarget(top_context["context"]))
+    recursive_compare({
+        'realm': any_string,
+        'exceptionDetails': {
+            'columnNumber': 0,
+            'exception': {
+                'handle': any_string,
+                'type': 'error'},
+            'lineNumber': 0,
+            'stackTrace': {
+                'callFrames': []},
+            'text': any_string}},
+        exception.value.result)
+
+
+@pytest.mark.asyncio
+async def test_exception(bidi_session, top_context):
+    with pytest.raises(ScriptEvaluateResultException) as exception:
+        await bidi_session.script.evaluate(
+            expression="throw Error('SOME_ERROR_MESSAGE')",
+            target=ContextTarget(top_context["context"]))
+
+    recursive_compare({
+        'realm': any_string,
+        'exceptionDetails': {
+            'columnNumber': 0,
```

Comment:
I don't know if we will be able to assert line/column numbers in a cross browser way. I get different values on Firefox both for this test and for the rejected promise one.

But we discuss that in a follow up issue.

Reply:
In that case, let's add an `any_number` helper for those, similar to the `any_string` one.

Reply:
agree. Removed line and column number assertion completely. Added `any_stack_trace` validator.